package.json

```diff
@@ -21,11 +21,42 @@
   "devDependencies": {
     "@octokit/types": "^6.17.2",
     "@types/jest": "^26.0.23",
+    "@typescript-eslint/eslint-plugin": "^4.28.1",
+    "@typescript-eslint/parser": "^4.28.1",
     "@vercel/ncc": "^0.28.6",
+    "eslint": "^7.29.0",
+    "eslint-config-haykam": "^1.10.0",
+    "eslint-plugin-extra-rules": "^0.0.0-development",
+    "eslint-plugin-html": "^6.1.2",
+    "eslint-plugin-jsdoc": "^32.3.4",
+    "eslint-plugin-json": "^2.1.2",
+    "eslint-plugin-markdown": "^2.2.0",
```

Comment:
why do we need html, jsdoc, json and markdown?

Reply:
Unfortunately, due to the limitations of ESLint, these dependencies are forced upon any project using a configuration that depends on it.